### PR TITLE
Remove shorthand.xml from the build process

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:
Remove shorthand.xml from the build process, because the tools now generate a proper XCCDF 1.2 file. We will achieve this removal by merging the responsibility of the `relabel_ids.py` to `build_shorthand.py`. Then, we will rename `build_shorthand.py` to `build_xccdf.py`.

For more details please read the commit messages of every commit.

#### Rationale:
Simplify the build process, avoid unnecessary writing and loading back, improve naming.

### CODE REVIEW

**Comment by jan-cerny** [comment]
I have fixed the Code Climate problem, and I have rebased this PR on the top of the latest upstream master branch.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the update. Can you please update the [build system docs](https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer/07_understanding_build_system.md) with the changes you made?

**Comment by jan-cerny** [comment]
I have updated the build system documentation and then I cleaned shorthand from the CMake files.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Just one minor change in the markdown. This should fix the Code Climate issues.

File: docs/manual/developer/07_understanding_build_system.md [hunk 1]
```diff
@@ -74,17 +74,16 @@ and testing targets. While the specifics should be understood from this file
...
-    and profiles) -- this is a single document that resembles the structure
-    of the XCCDF but isn't necessarily standards compliant,
- 3. Generation of templated content (such as Bash, OVAL, and Ansible) from
-    the metadata contained in the rule YAML files,
- 4. Transformation of the shorthand into unlinked and linked XCCDF documents,
- 5. Combining the various OVAL, OCIL, and XCCDF documents into a single
-    Datastream document,
- 6. Generating any derived products (such as CentOS and Scientific Linux).
-
+ 1. Generate SCE content and metadata.
```
[the remark this answers is not in the document]
Comment (jan-cerny) [reply docs/manual/developer/07_understanding_build_system.md:77]: I'm confused, it already uses `1.` here, so probably you wanted to say the opposite.

**Comment by jan-cerny** [comment]
I have [Change list to not use numbers

**Comment by Mab879** [comment]
I think what is tripping up Code Climate here is that some lists in this file have a space before the `-` and some don't. I believe that not having the space is preferred and what is enforced by Code Climate. 

To fix this issue, you may want to reformat the lists in `docs/manual/developer/07_understanding_build_system.md`.

File: docs/manual/developer/07_understanding_build_system.md [hunk 2]
```diff
@@ -74,55 +74,49 @@ and testing targets. While the specifics should be understood from this file
...
-    and profiles) -- this is a single document that resembles the structure
-    of the XCCDF but isn't necessarily standards compliant,
- 3. Generation of templated content (such as Bash, OVAL, and Ansible) from
-    the metadata contained in the rule YAML files,
- 4. Transformation of the shorthand into unlinked and linked XCCDF documents,
- 5. Combining the various OVAL, OCIL, and XCCDF documents into a single
-    Datastream document,
- 6. Generating any derived products (such as CentOS and Scientific Linux).
-
+- Generate SCE content and metadata.
```
Comment (Mab879) [inline docs/manual/developer/07_understanding_build_system.md:77]: Was it your intent to convert this to an unordered list? If not, you replace `-` with `1.`. Markdown will auto number them.

File: docs/manual/developer/07_understanding_build_system.md [hunk 3]
```diff
@@ -62,9 +62,9 @@ CMake requires projects have an entry point called `/CMakeLists.txt`. This
 uses the CMake language and drives building and installing the project. This
 file contains several things:
 
- 1. The many build-time options for customizing the types of content generated,
- 2. The hand-off for generating each product's specific content,
- 3. Common installation, testing, and distribution targets.
+- The many build-time options for customizing the types of content generated,
```
Comment (Mab879) [inline docs/manual/developer/07_understanding_build_system.md:65]: Was it your intent to convert this to an unordered list? If not, you replace `-` with `1.`. Markdown will auto-number them.

**Review by Mab879 — COMMENTED** [review]
Other than my two minor questions on the Markdown, everything looks good. Content builds just fine even with SCE enabled.

Thanks for the PR.

File: docs/manual/developer/07_understanding_build_system.md (hunk 2 above)
Comment (jan-cerny) [reply docs/manual/developer/07_understanding_build_system.md:77]: It wasn't but it's what I understood you wanted. :confused:

**Comment by jan-cerny** [comment]
@Mab879 Using `-` is intentional, I don't plan to renumber the lists in the docs.

**Review by Mab879 — APPROVED** [review]
Waving the code climate test as tets are coming in other PRs.

**Pull request merged**